src/core/tests/opset.cpp

```diff
@@ -74,8 +74,8 @@ INSTANTIATE_TEST_SUITE_P(opset,
                                          OpsetTestParams{ov::get_opset11, 177},
                                          OpsetTestParams{ov::get_opset12, 178},
                                          OpsetTestParams{ov::get_opset13, 186},
-                                         OpsetTestParams{ov::get_opset14, 189},
-                                         OpsetTestParams{ov::get_opset15, 7}),
+                                         OpsetTestParams{ov::get_opset14, 188},
+                                         OpsetTestParams{ov::get_opset15, 6}),
```

Comment:
```suggestion
                                         OpsetTestParams{ov::get_opset15, 8}),
```
To fix opsets tests

Reply:
Good catch, Thanks! That was a merge artifact after resolving conflicts:)